Makefile

```diff
@@ -0,0 +1,60 @@
+.DEFAULT_GOAL := help
+
+SHELL := /bin/bash
+VENV = .venv
+
+ifeq ($(OS),Windows_NT)
+	VENV_BIN=$(VENV)/Scripts
+else
+	VENV_BIN=$(VENV)/bin
+endif
+
+.PHONY: help
+help:  ## Display this help screen
+	@echo -e "\033[1mAvailable commands:\033[0m\n"
+	@grep -E '^[a-z.A-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "  \033[36m%-18s\033[0m %s\n", $$1, $$2}' | sort
+
+.PHONY: requirements
+requirements: requirements-r requirements-py requirements-rs ## Install all project requirements
+
+.PHONY: requirements-r
+requirements-r:
+	Rscript -e 'install.packages("pak", repos = sprintf("https://r-lib.github.io/p/pak/stable/%s/%s/%s", .Platform$$pkgType, R.Version()$$os, R.Version()$$arch))'
+	Rscript -e 'pak::repo_add(extendr = "https://extendr.r-universe.dev"); pak::local_install_deps(dependencies = c("all", "Config/Needs/dev", "Config/Needs/website"))'
+
+.venv:
+	python3 -m venv $(VENV)
+
+.PHONY: requirements-py
+requirements-py: .venv
+	$(VENV_BIN)/python -m pip install --upgrade pip
+	$(VENV_BIN)/pip install --upgrade mkdocs-material
+
+.PHONY: requirements-rs
+requirements-rs:
+	rustup toolchain install nightly
+	rustup default nightly
+
+.PHONY: build
+build: ## Compile polars R package and generate Rd files
+	Rscript -e 'rextendr::document()'
```

Comment:
I would expect this target or `build-all` to call `R CMD build .`

Reply:
That makes sense.

I expect the target we use most routinely to update README.md, and `build-all` was intended to be the target for that.
What name would you suggest? Simply `all`?

Reply:
Either that or `build-readme`, `build-pkg`, `build-doc` and then `build` or `build-all` does them all. `all` feels a bit too generic to me, but I don't have a strong view on this. Whatever you pick will be fine.

Reply:
The point here is that README can only be updated after the R package has been built.

If we expect that `build` to update README, we need to update README in `build` itself, not in the dependencies of `build`.

Perhaps we need to define a new target with dependencies on `build`, `test`, and `README.md`.

Reply:
Ah yes, I see. As I said, this is mostly dev-facing, so I don't think we need to worry too much about user interface or naming here. Whatever you think is best!